python/tk_multi_publish2/processing/plugin.py

```diff
@@ -200,7 +256,7 @@ def run_accept(self, item):
         """
         try:
             return self._plugin.accept(self.settings, item)
-        except Exception, e:
+        except Exception:
```

Comment:
Any reason why you didn't update this method to use the fancy new context manager? :P

Reply:
The error logging was different and it felt clunky to do

```
try:
    with ...
except:
   return {"accepted": False}
```